package.json

```diff
@@ -39,5 +39,12 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@faker-js/faker": "^7.6.0",
```

Comment:
Faker has been picked up by new maintainers after the previous maintainer went rouge and added malware. The library has been rebuilt and is safe. Read more [here](https://fakerjs.dev/about/announcements/2022-01-14.html)